spec/models/help_text_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe HelpText, type: :model do
```

Comment:
If we don't have any unit tests for this model (looking at the model I don't see any obvious ones) we can probs delete this file.

Reply:
Overlooked this, thanks for catching it 😄